src/index.html

```diff
@@ -11,8 +12,74 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <!-- #endregion -->
+
+  <body
```

Comment:
The `body` tag should not have the `class` or `data-qa` attributes. These are usually reserved for inner content elements.